influxdb-backup/backup.sh

```diff
@@ -0,0 +1,10 @@
+echo 'Backup Influx metadata'
```

Comment:
add `#!/bin/bash` at the beginning. Always specify your shell.

Although I've not done it in our stuff consistently, we ought to have a standard comment block at the beginning of each file.